pkg/api/controller.go

```diff
@@ -48,6 +49,7 @@ type Controller struct {
 	Server          *http.Server
 	Metrics         monitoring.MetricServer
 	CveInfo         ext.CveInfo
+	SyncOnDemand    SyncOnDemand
```

Comment:
This is going backwards, extensions is leaking into controller.

Reply:
like the above CveInfo and Metrics, right? :D

Reply:
everything :(